src/main/java/com/kornello/service/FeignPictureService.java

```diff
@@ -0,0 +1,13 @@
+package com.kornello.service;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+@FeignClient(value = "nasaPictures", url = "https://api.nasa.gov/")
+public interface FeignPictureService {
+    @GetMapping("/mars-photos/api/v1/rovers/curiosity/photos?api_key=6tttT1IDPRn64RM1YuKi6roj6yYforNpZ4eejugS&sol={sol}&camera={cam}")
```

Comment:
It would be better to add the apiKey as a @RequestParam to the method arguments rather than hard-coding it in the URL, as this value may change from time to time and you will need to rebuild and redeploy your service every time the key changes. rather than just calling this method with another parameter